appserver/admingui/payara-console-extras/src/main/resources/hazelcast/hazelcast.jsf

```diff
@@ -112,6 +112,9 @@
         <sun:property id="dynamic" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nhc.hazelcast.configuration.dynamic}"  helpText="$resource{i18nhc.hazelcast.configuration.dynamicHelp}">
             <sun:checkbox id="dynamic" selected="#{pageSession.dynamic}" selectedValue="true" />
         </sun:property>
+        <sun:property id="enableAutoPortSelection" labelAlign="left" noWrap="#{true}" overlapLabel="#{false}" label="$resource{i18nhc.hazelcast.configuration.enableAutoPortSelection}"  helpText="$resource{i18nhc.hazelcast.configuration.enableAutoPortSelectionHelp}">
```

Comment:
There's a discrepancy here between the admingui name and the domain.xml name. I prefer `PortAutoIncrement` to `AutoPortSelection`, but as long as it's consistent I don't mind either.

Reply:
`PortAutoIncrement` is the name of the HZ setting. The ticket asked for the parameter to be named `enableAutoPortSelection`. That is why. I found this the least confusing.

Reply:
Jumping on the back of this - I'd rename it as per my other comments to `Auto Increment Port` as it reads better.
I'm fairly confident the  parameter name given in the issue was just an example name, not necessarily a strict requirement.